std/array.d

```diff
@@ -818,6 +818,10 @@ private void copyBackwards(T)(T[] src, T[] dest)
     Inserts $(D stuff) (which must be an input range or any number of
     implicitly convertible items) in $(D array) at position $(D pos).
 
+    Params:
+        array = A reference to the array that $(D stuff) will be inserted into.
```

Comment:
This is slightly misleading, as this makes it sound like the caller needs to pass a reference to the array, when the function is the one taking it by reference.